src/knex-store.ts

```diff
@@ -112,6 +123,49 @@ function knex_store(this: any, options: Options) {
     }
   )
 
+  seneca.add('sys:entity,transaction:begin', async function(this: any, msg: any,reply: any) {
+      reply({
+        get_handle: () => ({ id: this.util.Nid(), name: 'knex' })
+      })
+  })
+
+  seneca.add(
+    'sys:entity,transaction:end',
+    async function (msg: any, reply: any) {
+      const transaction = msg.details()
+      const client = transaction.client
+
+      try {
+        const commit = await client.commit()
+
+        if (commit) {
+          reply({
+            done: true,
```

Comment:
we need to establish a standard set of messages and responses for transactions; this is a work in progress

Reply:
Today it helps me to debug

Reply:
OK, but please do keep in mind - If this makes it into the published version, it may prove difficult to remove it from future published versions due to backwards compatibility.

Reply:
yeah - and we need a type for this